tasks/main.yml

```diff
@@ -1,7 +1,14 @@
 ---
-- name: Install apt-transport-https
+- name: Install apt-transport-https and lsb-release
   apt:
-    name: apt-transport-https
+    name:
+      - apt-transport-https
+      - lsb-release # Needed because we use `ansible_lsb` below.
```

Comment:
Instead of using `ansible_lsb.id` and `ansible_lsb.codename` in the apt repo task in line 19, couldn't we just use `ansible_distribution | lower` and `ansible_distribution_release`, respectively? Then we wouldn't have to install anything.